plugins/modules/github_key.py

```diff
@@ -82,8 +82,14 @@
     name: Access Key for Some Machine
     token: '{{ github_access_token }}'
     pubkey: '{{ ssh_pub_key.stdout }}'
-'''
 
+# Alternatively, a single task can be used reading a key from a file
```

Comment:
I would add this:
```suggestion
# Alternatively, a single task can be used reading a key from a file on the controller
```
to make clear that the difference to the above tasks is that here the file is read on the controller, not on whatever machine ansible talks to (which could also be the controller, but doesn't have to be).